src/ch/imedias/rsccfx/view/RsccShowTokenView.java

```diff
@@ -102,7 +96,8 @@ private void initFieldData() {
   }
 
   private void bindFieldsToModel() {
-    //make the bindings to the modelit
+    //make the bindings to the model
+    // FIXME: only the last line binds things to the model, the rest belongs in layoutForm
```

Comment:
@Kennox only the last line binds things to the model, the rest belongs in layoutForm